### DISCUSSION TITLE
Chia transfer Delay

### DISCUSSION THREAD

**Discussion by myvole** [Chia-Network/chia-blockchain#6412] [Support]
Hi, I`m havint a trouble whith transferring chia from one walt to another.
From main Walet chia was sucssesfuly sent but not resived to the other walet.

![image](https://user-images.githubusercontent.com/85335614/120760718-1d3a1000-c51d-11eb-9f03-cd23672282aa.png)

![image](https://user-images.githubusercontent.com/85335614/120760807-3773ee00-c51d-11eb-9983-ee807279570f.png)

 Can some one help me! Tnx!

**Comment by XIII39** [comment]
Is your second wallet fully synced to the blockchain?
you can check https://www.chiaexplorer.com/blockchain/address/YOUR_WALLET_ADDRESS to see if it has been received.

**Reply by myvole** [reply]
My walet does not shown in chia explorer.

![image](https://user-images.githubusercontent.com/85335614/120767382-d00d6c80-c523-11eb-8601-000420ed3efb.png)

My walet is fully Synced
![image](https://user-images.githubusercontent.com/85335614/120767476-e4516980-c523-11eb-83d1-7f7445a1627e.png)
![image](https://user-images.githubusercontent.com/85335614/120767519-eddad180-c523-11eb-88a8-d025ec5f91ec.png)

**Reply by myvole** [reply]
My Walet Address xch1d6umydnpc33lq2g8tlfq3ll37ax0rm0caeq86lycz5fddd806ntqmluvlj

**Reply by myvole** [reply]
issue closed. tnx.

**Comment by BrandtH22** [comment]
@myvole , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
